chpl/chpl-service/src/main/java/gov/healthit/chpl/CHPLServiceConfig.java

```diff
@@ -263,6 +265,19 @@ public RestTemplate jiraAuthenticatedRestTemplate()
         CloseableHttpClient httpClient = HttpClients.custom().setSSLSocketFactory(csf).build();
         HttpComponentsClientHttpRequestFactory requestFactory = new HttpComponentsClientHttpRequestFactory();
         requestFactory.setHttpClient(httpClient);
+        int requestTimeout = DEFAULT_REQUEST_TIMEOUT;
+        String requestTimeoutProperty = env.getProperty("jira.requestTimeoutMillis");
+        if (!StringUtils.isEmpty(requestTimeoutProperty)) {
+            try {
+                requestTimeout = Integer.parseInt(requestTimeoutProperty);
+            } catch (NumberFormatException ex) {
+                LOGGER.warn("Cannot parse " + requestTimeoutProperty + " as an integer. "
+                        + "Using the default value " + DEFAULT_REQUEST_TIMEOUT);
+                requestTimeout = DEFAULT_REQUEST_TIMEOUT;
```

Comment:
It seems odd to set this variable to the DEFAULT in line 268, and then set it again here, but I don't know that it hurts. Just looks funny